### ISSUE TITLE
dnvm list -detailed does not align locations on linux

### ISSUE THREAD

**Issue by JunTaoLuo** [aspnet/dnvm#458]
On Windows `dnvm list -detailed` produces this:
![image](https://cloud.githubusercontent.com/assets/2030323/10229541/944e4178-682d-11e5-8f77-a626c703539e.png)

Note that the locations are aligned.

On Linux the same command produces this:

![image](https://cloud.githubusercontent.com/assets/2030323/10229556/b16fcc72-682d-11e5-9308-78ebcfe47ba7.png)

Note that the location is no longer aligned for default.

**Comment by muratg** [comment]
Ugh, ugly. Same problem on Mac

```
% dnvm list -detailed

Active Version              Runtime Architecture OperatingSystem Alias Location
------ -------              ------- ------------ --------------- ----- --------
       1.0.0-beta5          mono                 linux/osx             ~/.dnx/runtimes
       1.0.0-beta6-12234    coreclr x64          darwin                ~/.dnx/runtimes
       1.0.0-beta6-12234    mono                 linux/osx             ~/.dnx/runtimes
       1.0.0-beta6          mono                 linux/osx             ~/.dnx/runtimes
       1.0.0-beta7-12274    mono                 linux/osx             ~/.dnx/runtimes
       1.0.0-beta7-12275    mono                 linux/osx             ~/.dnx/runtimes
       1.0.0-beta7          clr     x86          win                   ~/.dnx/runtimes
       1.0.0-beta7          coreclr x64          darwin                ~/.dnx/runtimes
       1.0.0-beta7          mono                 linux/osx             ~/.dnx/runtimes
       1.0.0-beta8-15564    mono                 linux/osx             ~/.dnx/runtimes
       1.0.0-beta8-15613    mono                 linux/osx             ~/.dnx/runtimes
       1.0.0-beta8-15654    mono                 linux/osx             ~/.dnx/runtimes
  *    1.0.0-rc1-15779      coreclr x64          darwin          default ~/.dnx/runtimes
       1.0.0-rc1-15779      mono                 linux/osx             ~/.dnx/runtimes
```

**Comment by mholo65** [comment]
Location was originally aligned when implemented in #416 but broke in dnvm global install fc63c6c

However, it's now fixed in #467
